src/test/java/org/sagebionetworks/bridge/spring/controllers/StudyControllerTest.java

```diff
@@ -799,6 +803,88 @@ public void deleteStudy() throws Exception {
         verify(mockStudyService).deleteStudy("delete-study", Boolean.TRUE);
     }
     
+    @Test
+    public void getStudyMemberships() throws Exception {
+        StudyParticipant participant = new StudyParticipant.Builder()
+                .withEmail(EMAIL)
+                .withEmailVerified(true)
+                .withRoles(ImmutableSet.of(DEVELOPER))
+                .withSynapseUserId(SYNAPSE_USER_ID).build();
+        when(mockSession.getParticipant()).thenReturn(participant);
+        doReturn(mockSession).when(controller).getAuthenticatedSession();
+
+        Study studyA = createStudy("Study A", "studyA", false);
+        Study studyB = createStudy("Study B", "studyB", true);
+        Study studyC = createStudy("Study C", "studyC", true);
+        Study studyD = createStudy("Study D", "studyD", true);
```

Comment:
Maybe order the studies in a random order to verify that we are properly sorting the studies? (Here and in the next test as well.)

Reply:
I shuffled these around to verify they are being sorted.